WoWPro_Leveling/Wrath/Neutral/WOTLK_ZulDrak.lua

```diff
@@ -43,21 +43,21 @@ T That's What Friends Are For...|QID|12903|N|Back to Crusader MacKellar at Crusa
 T Making Something Out Of Nothing|QID|12901|M|25.3,64.0|N|To Engineer Reed.|
 T A Great Storm Approaches|QID|12912|M|25.3,64.0|N|To Engineer Reed.|
 A Gymer's Salvation|QID|12914|PRE|12912|M|25.3,64.0|N|From Engineer Reed.|
-K Banshees|QID|12914|N|Kill the banshees in The Dead Fields for their essence. They are floating so you will probably have to pull them to you.|QO|1|T|Banshee|
+K Banshees|QID|12914|M|26.34,60|N|Kill the banshees in The Dead Fields for their essence. They are floating so you will probably have to pull them to you.|QO|1|T|Banshee|
 C Gymer's Salvation|QID|12914|QO|2|N|Now head to the area around Ebon Watch and kill the Icetouched Earthragers.|M|14.26,73.60|
 f Ebon Watch|QID|12914|M|14.01,73.59|N|Get the flightpoint from Baneflight|
 T The Ebon Watch|QID|12884|M|14.12,73.79|N|To Stefan Vadu.|
 A Kickin' Nass and Takin' Manes|QID|12630|PRE|12884|M|14.1,73.8|N|From Stefan Vadu.|
 A An Invitation, of Sorts...|QID|12631|O|N|(UI Alert) From the Unliving Choker. You get this quest chain if you didn't complete the Drakuru quest chain in Grizzly Hills.|
+A Darkness Calling|QID|12633|O|N|From the Writhing Choker. You get this quest chain if you completed the Drakuru quest chain in Grizzly Hills.|U|38673|M|PLAYER|
```

Comment:
This needs a PRE instead of an O referencing the end of the the Drakuru quest chain.